packages/parser/test/parse.test.ts

```diff
@@ -1932,6 +1932,7 @@ font:
                     offsetX: { value: 0, unit: 'rem' },
                     offsetY: { value: 0.25, unit: 'rem' },
                     blur: { value: 0.5, unit: 'rem' },
+                    inset: false,
```

Comment:
I'm not sure if this is the best approach to make sure a `false` inset doesn't end up in the output or if is best to create an additional test.

Reply:
No I think this is right! If I remember correctly, we want to normalize/fill in all default values with the parser. Otherwise, all plugins/tools built on Terrazzo have to all remember what the defaults are, which isn’t fun for anybody.

Reply:
Got it. I moved the "hiding" of the `inset` decision from the parser to the css plugin.